tests/integration/test_streaming_e2e.py

```diff
@@ -106,6 +109,21 @@ def test_happiness(self):
         assert factory.client().get_treatment('pindon', 'split2') == 'off'
         assert factory.client().get_treatment('maldo', 'split2') == 'on'
 
+        # test if changeNumber is missing
+#        split_changes = make_split_fast_change_event(4)
```

Comment:
why commented code?

Reply:
The code does not check if the changeNumber is None, so this test will fail.
I wanted to check if we need to add checks to ignore the SPLIT_UPDATE and SPLIT_KILL events if the changeNumber value is missing.